### PR TITLE
Remove Deprecation Warning for Version and Update getServerVersion Methods

### PR DESCRIPTION
**Pull request by JT122406**

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java [hunk 1]
```diff
@@ -2233,7 +2233,7 @@ public static boolean joinRandomArena(Player p) {
 
     public static List<IArena> getSorted(List<IArena> arenas) {
         List<IArena> sorted = new ArrayList<>(arenas);
-        sorted.sort(new Comparator<IArena>() {
```
Comment (andrei1058) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:2236]: was that redundant? what's the reason here?

Comment (JT122406) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:2236]: In java 11 you don't need to put what type of array it is as at compile time it knows the type of array

Comment (JT122406) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/arena/Arena.java:2236]: So just makes the code look a bit cleaner but I can always add it back

**Comment by tomkeuper** [comment]
Just qurious, I saw you changed Arrays.assList() to List.of() but what does it change? Whats the benefit of changing?

**Comment by JT122406** [comment]
> Just qurious, I saw you changed Arrays.assList() to List.of() but what does it change? Whats the benefit of changing?

More efficient when there is one item in the list

**Review by tomkeuper — APPROVED** [review]

**Review by andrei1058 — APPROVED** [review]

**Pull request merged**